README.md

```diff
@@ -9,304 +9,9 @@
-
-### Check Environment
-
-Follow __[System Requirements](#using-python)__.
-
-### Set Up
-
-```sh
-# get latest code
-git clone https://github.com/microsoft/superbenchmark
-cd superbenchmark
-
-# install superbench
-python3 -m pip install -e .[dev,test]
-```
-
-### Lint and Test
-
-```sh
-# format code using yapf
-python3 setup.py format
-
-# check code style with mypy and flake8
-python3 setup.py lint
-
-# run all unit tests
-python3 setup.py test
-```
-
-### Submit a Pull Request
-
-Please install `pre-commit` before `git commit` to run all pre-checks.
-
-```sh
-pre-commit install
-```
-
-Open a pull request to main branch on GitHub.
-
-
-## Contributing
-
-### Contributor License Agreement
-
-This project welcomes contributions and suggestions.  Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
-
-When you submit a pull request, a CLA bot will automatically determine whether you need to provide
-a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions
-provided by the bot. You will only need to do this once across all repos using our CLA.
-
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
-For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
-contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
-
-### Contributing principles
-
-SuperBenchmark is an open-source project. Your participation and contribution are highly appreciated. There are several important things you need know before contributing to this project:
-
-#### What content can be added to SuperBenchmark
-
-1. Bug fixes for existing features.
-2. New features for benchmark module (micro-benchmark, model-benchmark, etc.)
-
-   If you would like to contribute a new feature on SuperBenchmark, please submit your proposal first. In [GitHub Issues](https://github.com/microsoft/superbenchmark/issues) module, choose `Enhancement Request` to finish the submission. If the proposal is accepted, you can submit pull requests to origin main branch.
-
-#### Contribution steps
-
-If you would like to contribute to the project, please follow below steps of joint development on GitHub.
-
-1. `Fork` the repo first to your personal GitHub account.
-2. Checkout from main branch for feature development.
-3. When you finish the feature, please fetch the latest code from origin repo, merge to your branch and resolve conflict.
-4. Submit pull requests to origin main branch.
-5. Please note that there might be comments or questions from reviewers. It will need your help to update the pull request.
+_Check SuperBench website for more details._
```

Comment:
What does Superbench website mean? Do we need to add a link?

Reply:
link is not ready